src/components/TodoList/TodoList.tsx

```diff
@@ -1,22 +1,73 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoUser } from '../../types/User';
+import { getUser } from '../../api';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todoList: Todo[];
+  setUserModal: (arg: null | TodoUser)=> void;
+  eyeMark: number;
+  setEyeMark: (arg: number) => void
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
+export const TodoList: React.FC<Props> = ({
+  todoList, setUserModal, eyeMark, setEyeMark,
+}) => {
+  const eyeButtonHandler = (todo: Todo) => {
+    setEyeMark(todo.id);
+
+    return getUser(todo.userId)
+      .then(user => {
+        return { ...user, todo };
+      })
+      .then(mutateUser => setUserModal(mutateUser));
```

Comment:
you may save user here, in local state

Reply:
I need to pass it parent, then to the modal. Save here? what for?